server/channels/app/session.go

```diff
@@ -106,15 +106,7 @@ func (a *App) GetSession(token string) (*model.Session, *model.AppError) {
 	if session == nil || session.Id == "" {
 		session, appErr = a.createSessionForUserAccessToken(c, token)
 		if appErr != nil {
-			detailedError := ""
-			statusCode := http.StatusUnauthorized
-			if appErr.Id != "app.user_access_token.invalid_or_missing" {
-				detailedError = appErr.Error()
-				statusCode = appErr.StatusCode
-			} else {
-				c.Logger().Warn("Error while creating session for user access token", mlog.Err(appErr))
-			}
-			return nil, model.NewAppError("GetSession", "api.context.invalid_token.error", map[string]any{"Token": token, "Error": detailedError}, "", statusCode)
+			return nil, model.NewAppError("GetSession", "api.context.invalid_token.error", map[string]any{"Token": token}, "", appErr.StatusCode).Wrap(appErr)
```

Comment:
I don't think returning the `detailedError` to the client is too helpful. I'd rather keep it safe and wrap it (which means we log the error in the server logs). Invalid sessions don't seem like an issue that end users would debug.